unit_test/test_stdout.txt

```diff
@@ -0,0 +1,40 @@
+ADMINS.md
```

Comment:
This will change with anything changing in this folder, so change the test command to be `echo something` to have predictable output.

Reply:
Added removal logic. seems no longer applicable given the new logic.